appendix.tex

```diff
@@ -9,6 +9,9 @@
 \appendix
 \chapter{Appendix Title}
 
+% Appendix sections, subsections etc should not appear in the table of contents. So set tocdepth to 0 and set it back to 2 at the end. Do this for each appendix.
```

Comment:
Can I ask that you break/reflow this long line, to be consistent with what's already here?

Reply:
Will do this shortly and get back to you!